api/v1alpha1/networkchaos_types.go

```diff
@@ -369,6 +378,83 @@ func (corrupt *CorruptSpec) ToNetem() (*chaosdaemonpb.Netem, error) {
 	}, nil
 }
 
+// BandwidthSpec defines details of bandwidth limit.
```

Comment:
"Bandwidth represents the detail about bandwidth control action"  and "BandwidthSpec defines details of bandwidth limit." , does the "details" here need to be unified with other places? I think other places write "detail".